test/FsAutoComplete.Tests.Lsp/paket.references

```diff
@@ -6,7 +6,7 @@ Expecto
 Expecto.Diff
 Microsoft.NET.Test.Sdk
 YoloDev.Expecto.TestSdk
-AltCover
+# AltCover
```

Comment:
Was getting annoyed at test times taking a long time due to this coverage hooking.  Need to revert.